rts/System/LogOutput.cpp

```diff
@@ -64,16 +63,15 @@ static spring::unordered_map<std::string, int> GetEnabledSections() {
 	#endif
 	#if !defined(DEBUG)
 	// Always show at least INFO level of these sections
-	enabledSections += "Sound:35,VFS:30";
+	enabledSections += "Sound:35,VFS:30,";
 	#endif
 	enabledSections += StringToLower(configHandler->GetString("LogSections"));
+	enabledSections += ",";
 #endif
 
 	if (getenv("SPRING_LOG_SECTIONS") != nullptr) {
 		// allow disabling all sections from the env var by setting it to "none"
-		envSections += getenv("SPRING_LOG_SECTIONS");
-		envSections = StringToLower(envSections);
-
+		std::string envSections = StringToLower(getenv("SPRING_LOG_SECTIONS"));
```

Comment:
Consider putting the getenv result in a var earlier, not just to avoid calling it twice but also to reduce usage of the magic string constant 
```cpp
const auto envLogSections = getenv("SPRING_LOG_SECTIONS");
if (envLogSections != nullptr) {
  // allow disabling all sections from the env var by setting it to "none"
  std::string envSections = StringToLower(envLogSections);
```

Reply:
Fixed